pkg/agent/openflow/pipeline.go

```diff
@@ -242,45 +242,64 @@ const (
 	hairpinMark      = 0b1
 	// macRewriteMark indicates the destination and source MACs of the
 	// packet should be rewritten in the l3ForwardingTable.
-	macRewriteMark = 0b1
-	cnpDropMark    = 0b1
+	macRewriteMark  = 0b1
+	cnpNotAllowMark = 0b1
```

Comment:
cnpNotPassedMark? Not sure it is really better.

Reply:
BTW, what is this mark used for?

Reply:
Is "NotAllow" supposed to be either "Drop" or "Reject"? I think we typically use the umbrella term "Deny" for that

Reply:
Yea it's either `Drop` or `Reject`. Changed.